### PR TITLE
Implement Hyperledger Fabric Peer chart

### PR DESCRIPTION
**Pull request by alexvicegrab**
Signed-off-by: Alejandro Vicente Grabovetsky <sasha@aid.technology>

**What this PR does / why we need it**:

Implements the last key chart needed for a Hyperledger Fabric Deployment, the HLF Peer

**Special notes for your reviewer**:

Chart requires setting three external secrets (a genesis block for the orderer, and the certificate and key of the Orderer organisation)

### CODE REVIEW

File: incubator/hlf-peer/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,161 @@
+apiVersion: apps/v1beta2
```
Comment (unguiculus) [inline incubator/hlf-peer/templates/deployment.yaml]: `apps/v1`

**Review by unguiculus — CHANGES_REQUESTED** [review]
Please follow our best practices for persistence:
https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md#persistence

I'd suggest you move the chart to stable directly.

**Comment by alexvicegrab** [comment]
Hi @unguiculus, thank you for reviewing the chart and suggesting improvements. Please review at your convenience!

**Comment by alexvicegrab** [comment]
Hi @unguiculus, thanks for accepting the companion charts hlf-ord, hlf-ca and hlf-couchdb. Would you like me to do any further improvements to this chart?

Thank you!

**Comment by alexvicegrab** [comment]
/assign @unguiculus 

PTAL :)

**Comment by alexvicegrab** [comment]
@unguiculus, PTLA, is there any further changes needed?

**Comment by alexvicegrab** [comment]
/assign @carrodher 

Carlos, would you be able to take a look and approve if Chart is OK?

File: stable/hlf-peer/templates/pvc.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
```
Comment (unguiculus) [inline stable/hlf-peer/templates/pvc.yaml:1]: Please follow our best practices for persistence. See https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#persistence

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/hlf-peer/templates/pvc.yaml (hunk 2 above)
Comment (alexvicegrab) [reply stable/hlf-peer/templates/pvc.yaml]: Updated, PTLA

Comment (alexvicegrab) [reply stable/hlf-peer/templates/pvc.yaml:1]: @unguiculus, updated, PTLA

**Review by unguiculus — APPROVED** [review]

**Comment by davidkarlsen** [comment]
@alexvicegrab Please check the tests - there seems to be some snags: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/helm_charts/6165/pull-charts-e2e/12547/?log#log (more info on tests in https://github.com/helm/charts/blob/master/test/README.md)

**Comment by alexvicegrab** [comment]
@davidkarlsen, @unguiculus, please LGTM if you agree with my updates.

**Pull request merged**